.github/renovate.json

```diff
@@ -20,35 +38,72 @@
         "^@typescript-eslint/",
         "^eslint-",
         "^eslint$"
-      ]
+      ],
+    },
+    {
+      "enabled": true,
+      "groupName": "GraphQL",
+      "matchManagers": ["npm"],
+      "matchPackagePatterns": ["^@graphql/", "^graphql$", "^graphql-"]
     },
     {
       "enabled": true,
       "groupName": "Jest",
       "matchManagers": ["npm"],
-      "matchPackagePatterns": ["^@types/jest$", "^jest", "^ts-jest$", "^jest-"]
+      "matchPackagePatterns": ["^@types/jest$", "jest", "^jest-", "-jest$"],
+    },
+    {
+      "enabled": true,
+      "groupName": "OpenTelemetry",
+      "matchManagers": ["npm"],
+      "matchPackagePatterns": ["^@opentelemetry/"]
     },
     {
       "enabled": true,
       "groupName": "Playwright",
       "matchManagers": ["npm"],
-      "matchPackagePatterns": ["^@playwright/", "^playwright$", "^playwright-"]
+      "matchPackagePatterns": ["^@playwright/", "^playwright$", "^playwright-"],
+    },
+    {
+      "enabled": true,
+      "groupName": "Prettier",
+      "matchManagers": ["npm"],
+      "matchPackagePatterns": ["^prettier"],
     },
     {
       "enabled": true,
       "groupName": "ProseMirror",
       "matchManagers": ["npm"],
-      "matchPackagePatterns": ["^prosemirror-"]
+      "matchPackagePatterns": ["^prosemirror$"]
```

Comment:
Was this change intentional or did you revert the wrong pattern?

Reply:
We have some packages that end in `prosemirror`, as well as start, so I thought this would be an improvement.